tests/SelfTest/UsageTests/Matchers.tests.cpp

```diff
@@ -1081,6 +1081,7 @@ struct MatcherWasMovedOrCopied : std::exception {
 
 struct ThrowOnCopyOrMoveMatcher : Catch::Matchers::MatcherGenericBase {
     ThrowOnCopyOrMoveMatcher() = default;
+    // NOLINTNEXTLINE(bugprone-copy-constructor-init)
```

Comment:
I am looking at the docs, and apparently changing the code below to `Catch::Matcher::MatcherGenericBase(other)` would work.

Or maybe deleting the initialization of the base class completely might work, as the argument is already marked as unused via deleted name.

Reply:
Deleting the call would trigger a different warning :smile: Calling the copy constructor _might_ work, though. I just find it all mightily stupid.

Reply:
Calling the copy constructor worked. I have removed the suppressions.